### DISCUSSION TITLE
TLS中转节点连接失败

### DISCUSSION THREAD

**Discussion by jelly0925** [SagerNet/SagerNet#135] [Q&A] [answered]
因为中转机使用的IP地址访问，验证不了证书造成节点连接失败，能不能添加 allowinsecure 跳过证书验证参数呢，谢谢
![image](https://user-images.githubusercontent.com/3022023/121484316-ae1f5880-ca01-11eb-8b9d-733759feb145.png)

**Comment by nekohasekai** [comment, marked as answer]
添加 SNI 项、证书或固定证书链。
